### DISCUSSION TITLE
关于macos的守护进程

### DISCUSSION THREAD

**Discussion by xiaowang7212** [AlistGo/alist#3683] [General]
使用任意方式编辑 ~/Library/LaunchAgents/ci.nn.alist.plist 并添加如下内容，修改 path_alist 为 AList 所在的路径，path/to/working/dir 为 AList的工作路径
以上是官方文档给出的方法，都是系统资源库的文件没法修改，并且不能新建文件ci.nn.alist.plist文件无法放进去，
path/to/working/dir 为 AList的工作路径 这个工作路径具体是哪 干嘛用的 · ··

**Comment by asygr** [comment]
用系统自带应用自动操作可以

**Reply by xiaowang7212** [reply]
请问 具体怎么弄呢
